.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "protobufs"]
 	path = protobufs
-	url = https://github.com/meshtastic/protobufs.git
```

Comment:
I think need revert back to original protobufs

Reply:
Done.
https://github.com/fzellini/meshtastic-firmware/blob/trackerd/.gitmodules